packages/mdc-textfield/_mixins.scss

```diff
@@ -612,3 +703,9 @@
     @include mdc-floating-label-ink-color($color);
   }
 }
+
+@mixin mdc-text-field-invalid_ {
```

Comment:
it looks like on line 245 there is already a mixin named `mdc-text-field-invalid_`. I think probably leads to unintentional changes.